packages/table/src/cell/cell.tsx

```diff
@@ -8,8 +8,9 @@
 import * as classNames from "classnames";
 import * as PureRender from "pure-render-decorator";
 import * as React from "react";
+import * as Classes from "../common/classes";
 
-import { Classes, IIntentProps, IProps } from "@blueprintjs/core";
+import { Classes as BlueprintClasses, IIntentProps, IProps } from "@blueprintjs/core";
```

Comment:
I could have sworn I was following an example I found somewhere else, but I can't seem to find it. If this is preferred I can change it.